ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -746,6 +793,15 @@ void HardwareInterface::doSwitch(const std::list<hardware_interface::ControllerI
         {
           cartesian_forward_controller_running_ = false;
         }
+        if (resource_it.hardware_interface == "ros_controllers_cartesian::TwistCommandInterface")
+        {
+          twist_controller_running_ = false;
+        }
+        if (resource_it.hardware_interface == "ros_controllers_cartesian::PoseCommandInterface")
+        {
+          ROS_INFO_STREAM("Stopping pose controller");
```

Comment:
This seems to be the only controller type that gets an info output. Is this intention?

Reply:
That seems to be a leftover from development. I will remove that.